devnet-sdk/system2/cluster.go

```diff
@@ -0,0 +1,40 @@
+package system2
+
+import "github.com/ethereum-optimism/optimism/op-supervisor/supervisor/backend/depset"
+
+type L2ClusterID genericID
+
+func (id L2ClusterID) String() string {
+	return genericID(id).string("L2Cluster")
+}
+
+func (id L2ClusterID) MarshalText() ([]byte, error) {
+	return genericID(id).marshalText("L2Cluster")
+}
+
+func (id *L2ClusterID) UnmarshalText(data []byte) error {
+	return (*genericID)(id).unmarshalText("L2Cluster", data)
+}
+
+type Cluster interface {
```

Comment:
What is `Cluster` in this case?

Reply:
Not all chains that share a L1 OPCM superchain deployment (same permissions etc.) may be interoperable yet. A cluster is the subset of chains that are coupled with interop.

Reply:
Could we add a small comment for the key types, such as this one?
This wasn't known/clear to me either.

Reply:
In the base PR there are docs + a typing for the kinds